pom.xml

```diff
@@ -9,9 +11,9 @@
     <description>JSON Reference for Java with Jackson.</description>
 
     <properties>
-        <github.user>adjohnson916</github.user>
+        <github.user>slavaschmidt</github.user>
```

Comment:
@slavaschmidt Can you restore `github.user` and `slasch` to me? If you need to keep yours for your bintray, maybe maintain two branches on your repo - e.g. a new one for your changes and `master` for this PR?